### PR TITLE
Looping and accent customisation, also improved length limiting

### PR DESCRIPTION
**Pull request by jayc331**

### CODE REVIEW

File: video_creation/final_video.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def make_final_video(number_of_clips):
     image_concat.audio = audio_composite
     final = CompositeVideoClip([background_clip, image_concat])
     final.write_videofile(
-        "assets/final_video.mp4", fps=30, audio_codec="aac", audio_bitrate="192k"
+        "output/" + name + ".mp4", fps=30, audio_codec="aac", audio_bitrate="64k"
```
Comment (JasonLovesDoggo) [inline video_creation/final_video.py]: why the lowering of the audio bitrate?

Comment (jayc331) [reply video_creation/final_video.py]: To reduce file sizes and time to write file.

**Comment by JasonLovesDoggo** [comment]
These have all been implemented in the newer versions

**Comment by jayc331** [comment]
Simply to lower the file size and reduce the file write time. It was a test really, I just forgot to change it back.

**Pull request closed without merging**